docs/en/maintain/backup.md

```diff
@@ -0,0 +1,202 @@
+# Backup and restore
+
+## High Availability Description
+
+* The nameserver cannot be fully hung up. Failover and recovery cannot be done if all hangs up
+* High availability is not guaranteed in the case of a single copy
+* In the case of two replicas and two nodes, the tablet can only hang on one node at most
+* In the case of two replicas and multiple nodes, the two tablet nodes where the same shard is located cannot hang at the same time
+* In the case of three replicas and three nodes, hanging a tablet node can automatically perform failover and data recovery. Hanging two tablet nodes does not guarantee automatic failover and data recovery, but reads and writes can return to normal within minutes
+* In the case of three replicas and multiple nodes, if the two tablets where the same shard is located hangs, the automatic failover and data recovery of the shard are not guaranteed, but the read and write can return to normal within minutes.
+* If there is a leader fragment in the hanging node and there is always write traffic, a small amount of data may be lost
+
+## Downtime and Recovery
+
+OpenMLDB high availability can be configured in automatic mode and manual mode. In automatic mode, if the node fails and recovers, the system will automatically perform failover and data recovery, otherwise manual processing is required
+
+The mode can be switched by modifying the auto_failover configuration. The default is to open the automatic mode. You can get the configuration status and modify the configuration in the following ways
+
+```bash
+$ ./bin/openmldb --zk_cluster=172.27.128.31:8090,172.27.128.32:8090,172.27.128.33:8090 --zk_root_path=/openmldb_cluster --role=ns_client
+> confget
+  key                 value
+-----------------------------
+  auto_failover       true
+> confset auto_failover false
+set auto_failover ok
+>confget
+  key                 value
+-----------------------------
+  auto_failover       false
+```
+
+**If auto_failover is enabled, if a node goes offline, the is_alive status of showtable will change to no. If the node contains a leader of a shard, the shard will re-select the master**
```

Comment:
**When auto_failover is enabled, if a node goes offline, the `is_alive` status of the `showtable` command will change to `no`. If the node contains a leader of a partition, the partition will re-select the leader. **